episodes/05-history.md

```diff
@@ -309,7 +320,7 @@ here's how Git works in cartoon form:
 If you read the output of `git status` carefully,
 you'll see that it includes this hint:
 
-```output
+```bash
```

Comment:
I think this should still be `output`: it is an excerpt from the output of `git status` containing a command, rather than a Bash command itself.

Reply:
```suggestion
```output
```